.github/workflows/knownprojects_build.yml

```diff
@@ -17,41 +17,42 @@ jobs:
       run:
         shell: bash
         working-directory: ./products/knownprojects
-    services:
-      postgis:
-        image: postgis/postgis:15-3.3-alpine
-        env:
-          POSTGRES_PASSWORD: postgres
-        options: >-
-          --health-cmd pg_isready
-          --health-interval 10s
-          --health-timeout 5s
-          --health-retries 5
-        ports:
-          - 5432:5432
     env:
-      BUILD_ENGINE: postgresql://postgres:postgres@postgis:5432/postgres
+      BUILD_ENGINE_DATABASE: ${{ secrets.SQL_ENGINE_EDM_DATA_SERVER }}/kpdb
+      BUILD_ENGINE_SCHEMA: build_${{ github.event.pull_request.number || github.ref_name }}
       EDM_DATA: ${{ secrets.EDM_DATA }}
       AWS_S3_ENDPOINT: ${{ secrets.DO_S3_ENDPOINT }}
       AWS_ACCESS_KEY_ID: ${{ secrets.DO_ACCESS_KEY_ID }}
       AWS_SECRET_ACCESS_KEY: ${{ secrets.DO_SECRET_ACCESS_KEY }}
     steps:
       - uses: actions/checkout@v3
 
+      - name: Set build environment variables
+        run: |
+          # # replace dashes with underscores to create a valid postgres schema name
+          BUILD_ENGINE_SCHEMA=$(echo ${BUILD_ENGINE_SCHEMA} | tr - _)
+          echo "BUILD_ENGINE_SCHEMA=$BUILD_ENGINE_SCHEMA" >> "$GITHUB_ENV"
+          # set postgres schema search path to prioritize BUILD_ENGINE_SCHEMA
+          BUILD_ENGINE=${BUILD_ENGINE_DATABASE}?options=--search_path%3D${BUILD_ENGINE_SCHEMA},public
+          echo "BUILD_ENGINE=$BUILD_ENGINE" >> "$GITHUB_ENV"
+
       - name: Setup build environment
         working-directory: ./
         run: ./bash/docker_container_setup.sh
 
-      - name: 1. dataloading ..
+      - name: dataloading ..
         run: ./bash/01_dataloading.sh
 
-      - name: 2. build ...
+      - name: build ...
         run: ./bash/02_build.sh
-      
-      # TODO upload outputs to a private S3 bucket
-      - name: 3. export ...
+
+      # TODO fix SCA aggregation
```

Comment:
What's the thought moving forward here? Get it building without this in main for now so enhancements can be worked on? But we'll still need this as part of the update with the enhancements I'm assuming

Reply:
Wasn't done in carto, we ran it as part of the last build